Article/Article.js

```diff
@@ -1,20 +1,22 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(articles) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.articles = articles;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.articles.querySelector(".expandButton");
+    // console.log(this.expandButton);
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = "expand";
+    // console.log(this.expandButton);
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener("click", this.expandArticle.bind(this));
```

Comment:
totally valid but don't be scared of that arrow function syntax so you don't have to bind `this`
`this.expandButton.addEventListener('click', () => this.expandArticle());`